src/CLI/clitree/cli-xml/ipv6.xml

```diff
@@ -24,6 +24,91 @@ limitations under the License.
     xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema
                         http://www.dellemc.com/sonic/XMLSchema/clish.xsd">
 
+<VIEW name="enable-view">
+    <COMMAND
+        help="show IPv6 commands"
+        name="show ipv6"
+        >
+    <ACTION builtin="clish_nop"/>
+    </COMMAND>
+<!--====================show ipv6 neighbors <ip-addr> commands ======================-->
+    <COMMAND
+        help="Display IPv6 neighbor information"
+        name="show ipv6 neighbors">
+        <PARAM
+            name="nbr-subcommands"
+            help="Neighbor subcommands"
+            mode="switch"
```

Comment:
Same is the case for IPv6 as well, do we need this PARAM with mode="switch", and no need of optional="true"

Reply:
ok